src/gromit-mpx.h

```diff
@@ -67,6 +67,7 @@ typedef struct
   guint           width;
   gfloat          arrowsize;
   guint           minwidth;
+  guint           maxsize;
```

Comment:
Please call this minwidth to be in line with the other parameter.

Reply:
I assume you meant maxwidth? Since minwidth is already declared.

Reply:
If you meant maxwidth, then I did as you wanted. If not, then please notify me.